front/src/Phaser/Game/GameMap.ts

```diff
@@ -165,4 +173,47 @@ export class GameMap {
         }
     }
 
+    private putTileInFlatLayer(index: number, x: number, y: number, layer: string): void {
+        const fLayer = this.findLayer(layer);
+        if ( fLayer == undefined ) {
+            console.error("The layer that you want to change doesn't exist.");
+            return;
+        }
+        if (fLayer.type !== 'tilelayer') {
+            console.error("The layer that you want to change is not a tilelayer. Tile can only be put in tilelayer.");
+            return;
+        }
+        // @ts-ignore
```

Comment:
check fi its not a string before then you dont need tsignore
    if (fLayer.data instanceof Array) {